backend/src/test/java/org/eclipse/tractusx/semantics/hub/ModelsApiTest.java

```diff
@@ -158,10 +158,23 @@ public void testGenerateOpenApiEndpointSpecExpectSuccess() throws Exception {
       ;
    }
 
+  @Test
+  public void testAasxEndpointExpectSuccess() throws Exception {
+    String urnPrefix = "urn:bamm:org.eclipse.tractusx.testaas:1.0.0#";
+    mvc.perform(post( TestUtils.createValidModelRequest(urnPrefix),"DRAFT") )
+        .andDo( MockMvcResultHandlers.print() )
+        .andExpect( status().isOk() );
+    
+    mvc.perform(MockMvcRequestBuilders.get("/api/v1/models/{urn}/aas?aasFormat=xml", toMovementUrn(urnPrefix)).with(jwtTokenFactory.allRoles()))
+      .andDo( MockMvcResultHandlers.print())
```

Comment:
A test for `file`is missing.

Also a test for  an unsupported format like `json` would be great.